docs/en/data_types/domains/IPv4.md

```diff
@@ -0,0 +1,66 @@
+## IPv4
+`IPv4` is a domain based on `UInt32` type, this is basically a typed replacement of `UInt32` for storing IPv4 values. It provides compact storage (compared to `String`), human-friendly output (compared to `UInt32`) and column type information on inspection.
```

Comment:
IPv4 doesn't look as `SQL domain`. It looks like usual embedded data type, while domain is a custom entity, that user define within their database with `CREATE DOMAIN` clause.

Reply:
Yep, I've mention that in the `overview.md`, `IPv4` and `IPv6` are kind of builtin domains, while there is still much work to do in order to support full-fledged user-defined domains.